apps/voting-disputable/test/contracts/voting_settings.js

```diff
@@ -209,4 +210,54 @@ contract('Voting settings', ([_, owner, anyone, holder51, holder20, holder29]) =
       })
     })
   })
+
+  describe('changeExecutionDelay', () => {
+    context('when the sender is allowed', () => {
+      const from = owner
+
+      context('when the new window is valid', () => {
+        const newDelay = DAY
```

Comment:
This is the same as the old one, right? We should add test for real changes of the value.

Reply:
Yea! Good catch!